css-canvas-medical/index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div id="container">
+    <div class="row">
+      <div class="column">
+        <img class="logo" src="images/logo-blue.svg">
+      </div>
+      <div class="column">
+        <p class="navbar-text">Product</p>
+        <p class="navbar-text">Implementation</p>
+        <p class="navbar-text">About Us</p>
+        <p class="navbar-text">Blog<p>
+        <button class="button">Contact Us</button>
+      </div>
+    </div>
+    <hr id="navbar-line">
+    <br>
+    <br>
+    <br>
```

Comment:
No `br` elements, if you need additional space, you should use `margin` or `padding`